develop/tutorials/articles/100-tooling/04-liferay-workspace/06-managing-themes-in-liferay-workspace.markdown

```diff
@@ -0,0 +1,95 @@
+# Managing Themes in Liferay Workspace
+
+Creating a @product@ theme can be accomplished using several different tools:
+
+- [Theme Generator](/develop/tutorials/-/knowledge_base/7-1/creating-themes)
+  (Node.js-based)
+- [Project template/archetype](/develop/reference/-/knowledge_base/7-1/theme-template)
+  (Gradle/Maven-based)
+- etc.
+
+Liferay Workspace offers an environment where developers with differing theme
+development expertise can collaborate and work together in one place. For
+example, suppose Tony theme developer is a guru at Gradle-based themes and Tracy
```

Comment:
So I think the messaging should be

Theme developers should use Lifeary Theme SDK (node.js tasks).  But the story for Liferay Workspace is that those themes build by node can be safely integrated into their overalll DevOps story with Liferay workspace because workspace has a "themes" folders that can build those node.js / liferay themes sdk based themes.  

I don't think we want to draw the picture of a Theme developer who would want to choose gradle-based theme-builder(java) to build the themes and one that would choose our Node based development.  everything we are pushing is for node based and for workspace users they put those node-based themes in "themes" folder and everything "just works".

The themes in the 'wars' folder using ThemeBuilder is for those teams who want to build themes in the traditional way and use pure java to build them (i.e. they can't use node.js tools to build their theme in their CI environment).

Reply:
@gamerson can you let me know if the new explanation I provided is sufficient? Or if there are other details you think we should point out?

Reply:
That looks good!